runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
On Tue, Sep 29, 2015 at 04:18:53PM -0700, Alexander Morozov wrote:
> > +It can be used for notifying user that real process is spawned.
> 
> Fun thing, that actually you don't need process to be started to
> join container, so use-case is invalid :)

I need to know that the container exists ;).  If the container
includes a PID namespace (or anything else that's bound to the
container process, the join has to be launched after those bits have
been created.  With the lifecycle in [1], that means anything after
step 3, which means that the last hook in a pre-start hook chain would
be a reasonable place for my Nginx stuff and the garden stuff too
(sorry I didn't notice that earlier).

Which gets me back to wondering what post-start is good for :p.  Can
you elaborate on why the last hook in a pre-start hook chain wouldn't
work for your “is ready” notification [2], but a post-start hook
would?  I think my problem is figuring out what the interesting change
has been, since “we've changed the binary that the application is
running” doesn't seem like a particularly actionable event.

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/frUXLljXy8Y/_sg-BidnAwAJ
[2]: https://github.com/opencontainers/specs/pull/202#discussion_r40725147